### PR TITLE
[PS-2022] Add autofill values for german websites

### PR DESCRIPTION
**Pull request by gleichda**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other
```

## Objective

Add a few german auto fill field names

## Code changes

- **apps/browser/src/services/autofillConstants.ts** Add autofill values for german websites

### CODE REVIEW

File: apps/browser/src/services/autofillConstants.ts [hunk 1]
```diff
@@ -312,7 +312,10 @@ export class IdentityAutoFillConstants {
     "familienname",
   ];
 
-  static readonly EmailFieldNames: string[] = ["e-mail", "email-address"];
+  static readonly EmailFieldNames: string[] = [
+    "e-mail", 
+    "email-address",
+  ];
```
Comment (djsmith85) [inline apps/browser/src/services/autofillConstants.ts:318]: Really minor thing: Our linter might complain about this. It's usally one line until it reaches a certain length and then it wraps onto the next lines, lke the other Constants

**Review by djsmith85 — APPROVED** [review]
@gleichda Thank you for your contribution 🎉  I have reviewed and tested your changes, and they indeed improve identity-autofill on German sites.

I initially had a hard time finding sites I could use for testing. Testing autofill is unfortunately still a mostly manual process. I tested this on one of the contact forms of the [Deutsche Rentenversicherung](https://www.deutsche-rentenversicherung.de/DRV/DE/Beratung-und-Kontakt/Kontakt/Kontaktformular/kontaktformular_node.html). Nothing more German than that, besides maybe other Government websites.

Great example I found, which most autofill will have trouble with given those input field names ([Finanzamt/Elster](https://www.elster.de/eportal/wizard/seq/nichtsteuerlichenachricht-4/eingabe) 🤦‍♂️).

Your changes would be ready to get passed onto QA, but I think we could take this a step further. Adding more of the missing German input field names. I have already identified and locally tested some additions.

Would you be interested helping me with adding, but also testing/verifying, additional changes?

**Comment by djsmith85** [comment]
Instead of looking for individual sites that fit our test-cases, maybe it makes sense to create a German identity autofill page with all possible combinations taken from the `autofillConstants.ts`

Bitwarden source field as header/category 
  and then all fields matching it per constant

### titles
  - titel, anrede,
### names
  - name, vorname, nachname
### city
  - ort, wohnort, stadt
### zipcode
  - plz, postleitzahl
- etc.

This would also make it easier for QA to vet if all fields got filled without knowing what each fieldname translates to.

**Comment by gleichda** [comment]
hi @djsmith85 ,
happy to contribute more but unfortunately I'm definitely not a Javascript expert. So I'd need some more guidance

**Comment by djsmith85** [comment]
@gleichda Thanks again for the contribution. I have merged in the newest `master` and have included my addition to your branch. Once the Vault/Autofill-Team has reviewed our changes, I'll be moving this to QA. 

I unfortunately haven't had any time to extend on my idea of a dedicated auto-fill test site.

**Review by cagonzalezcs — APPROVED** [review]
Pulled this project down locally and tested auto-fill behavior within the two websites mentioned by @djsmith85. Likely there are more improvements to make here, especially in consideration of other languages and their unique identifiers for input elements.

That said, this does present an improvement on our current implementation. Approving this implementation as a representative of the Autofill team, and will keep in mind improvements for the future.

**Comment by djsmith85** [comment]
@gleichda QA has approved the changes and they are ready for merging. They will be included in a future release.

**Pull request merged**